### PR TITLE
Update badges and add long description to pypi

### PR DESCRIPTION
**Pull request by GeorgianaElena**
* Adds badges according to the team compass page https://jupyterhub-team-compass.readthedocs.io/en/latest/building-blocks/readme-badges.html
* Updates the project description on pypi.

(cc: @consideRatio @manics :sun_with_face: )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+[![TravisCI (.org) build status](https://img.shields.io/travis/org/jupyterhub/firstuseauthenticator/master?logo=travis)](https://travis-ci.org/jupyterhub/firstuseauthenticator)
```
Comment (consideRatio) [inline README.md]:
```suggestion
[![TravisCI (.org) build status](https://img.shields.io/travis/jupyterhub/firstuseauthenticator/master?logo=travis)](https://travis-ci.org/jupyterhub/firstuseauthenticator)
```

**Comment by GeorgianaElena** [comment]
Sure, why not. I'll reorder them :heart:

**Comment by GeorgianaElena** [comment]
Thanks @consideRatio :heart:

**Comment by consideRatio** [comment]
Thanks @GeorgianaElena! :heart:

**Pull request merged**